src/test/java/com/bitheads/braincloud/services/TestFixtureBase.java

```diff
@@ -49,72 +49,84 @@ static String getServerUrl() {
     /// </summary>
     @BeforeClass
     public static void getIds() {
-        if (m_serverUrl.length() > 0)
```

Comment:
I think this change will break Jenkins Java Unit testing to run. It needs to read in the ID's before testing.